samples/grpc/restaurant-drone-deliveries-service-java/pom.xml

```diff
@@ -41,6 +41,11 @@
             <name>Akka library repository</name>
             <url>https://repo.akka.io/maven</url>
         </repository>
+    <repository>
+      <id>akka-snapshot-repository</id>
+      <name>Akka library snapshot repository</name>
+      <url>https://repo.akka.io/snapshots</url>
+    </repository>
```

Comment:
Keep the snapshot repository?

Reply:
Ups, must have been a local change I had but didn't notice, I'll remove it.